src/Tests/Equinor.Procosys.Library.Query.Tests/GetAllResponsibles/GetAllResponsiblesQueryTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Equinor.Procosys.Library.Query.GetAllResponsibles;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Equinor.Procosys.Library.Query.Tests.GetAllResponsibles
+{
+    [TestClass]
+    public class GetAllResponsiblesQueryTests
+    {
+        [TestMethod]
+        public void Constructor_SetsProperties()
```

Comment:
fine for me, but in further PBI's we dont have to create these simple tests on Constructor. Your choice. 
However, the test below, testing exception when no plant given is nice to have